### ISSUE TITLE
Small red mark in leaves having uncertainty in TreeADDs

### ISSUE THREAD

**Issue by JorgeRicoVivas** [OpenMarkov/root#253]
Originally reported by: **Manuel Luque (Bitbucket: [Manuel Luque](https://bitbucket.org/Manuel Luque), GitHub: Unknown)**

----------------------------------------

When visualizing a TreeADD, it could be useful to have a small red mark in leaves having uncertainty associated, similarly to tables.

----------------------------------------
- Bitbucket: https://bitbucket.org/cisiad/org.openmarkov/issue/253

**Comment by JorgeRicoVivas** [comment]
Wasn't this already done? This issue might have been solved, but not closed.

**Comment by fjdiez** [comment]
I agree with Manuel Luque: it would useful to implement this feature.
